docs/source/status_indent.rst

```diff
@@ -0,0 +1,5 @@
+.. _status_indent:
+
+ .. note::
+
+    You can add additional indent after ``└─`` and before branch name by setting git config key ``machete.status.branch.indent.len``.
```

Comment:
I'd say this is misleading, as the indent is in fact **2** even currently (with `machete.status.branch.indent.len` being effectively 0). Maybe sth around `extraSpace` would be a better name?